layouts/shortcodes/iframe.html

```diff
@@ -0,0 +1,5 @@
+{{ $url := .Get "url" }}
+<div style="width: 100%; height:calc(100vh - 200px)">
+    <iframe src="{{ $url }}" height="100%" width="100%"><a href="{{ $url }}">API Documentation</a></iframe>
```

Comment:
The intent was to provide a link to the documentation in the event the users browser did not support iframes.

Options:
- change the shortcode title to reflect the API Documentation nature
- generify the link text
- remove the link entirely